src/plugins/content_management/public/registry/content_type_details.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { CrudClient } from '../crud_client';
+
+/**
+ * Content type definition as it is registered in the content registry.
+ */
+export interface ContentTypeDetails {
```

Comment:
Follow up on my previous comment about naming, this would then be named something like `ContentMetadata`.

"...Details" made me think initially that it was the content details fetched (a single content from the DB). Which it is not, those are metadata about a content.